### ISSUE TITLE
[Request] Icon fonts instead of emojis?

### ISSUE THREAD

**Issue by Barbarossa93** [xwmx/nb#98]
I'm trying out the latest git branch of `nb` and I have to say that the folder feature is amazing; It's exactly what I need! I've noticed though that it uses emojis to symbolize whether an entry is a folder. Looking at your README, it seems like you have emojis for indicating other things as well. Personally, I'm not the biggest fan of emojis and instead prefer to use icon fonts such as [nerd fonts](https://github.com/ryanoasis/nerd-fonts). Would it be possible to implement an option to choose which style of symbols to use, or alternatively, options to use w/e symbol I prefer for each individual status type?

**Comment by hellquist** [comment]
Oh! Could it be a lack of emojis that is behind my pain for the last couple of hours? I have been installing nb and trying to get them little images to work for a couple of hours now, with no luck (Manjaro/XFCE). I have installed "everything" the page lists including everything that is "optional". I've swapped fonts. I've tried various terminals. I notably have NOT installed any type of emojis though...could that be it? :D

**Comment by xwmx** [comment]
@Barbarossa93 Glad you like it! These can be customized by assigning a different character or empty string to the relevant environment variable, which can be added to `~/.nbrc`. I just added some information about this to the README: https://github.com/xwmx/nb#indicators If your terminal font is using a nerd font, you should be able to assign the indicator variable to the unicode character that's used in the font for the desired glyph. A space is added between the character and the file information text. I haven't experimented with it outside of emoji characters yet.

@hellquist Probably. Maybe try installing and using the [noto-fonts-emoji](https://archlinux.org/packages/extra/any/noto-fonts-emoji/) package.

**Comment by Barbarossa93** [comment]
Wow, that was quick! Thank you so much! With folders at least, everything looks good/is working so far. I'll close this since my issue is technically solved.

**Comment by hellquist** [comment]
> @hellquist Probably. Maybe try installing the [noto-fonts-emoji](https://archlinux.org/packages/extra/any/noto-fonts-emoji/) package and see if that makes them available.

Ah, perfect. That did the trick! Now I have every helper tool there ever was so I should probably work out how to use those too! :D But first for making plugins actually work without having to sudo everything all the time. :)

**Comment by Barbarossa93** [comment]
Hey @xwmx, I just have a quick question. Are you planning on implementing a way to look within folders when using `nb` or `nb ls`? I just had this idea where the `ls` subcommand could have the `--tree|-t` option so that a tree is printed (probably using `tree` or `exa --tree`) so that all of the contents of the folders are easily visible.

**Comment by xwmx** [comment]
@Barbarossa93 Indeed, I've spent some time thinking about and experimenting with that. I just added a partial, undocumented, untested implementation that only works with a subset of other `list` options for now. The option is available on `list` as `nb list --tree` and `nb list --recursive`, and can also be used as `nb --tree` and `nb notebook:folder/path/ --tree`. I'll circle back and fix it up when I can.
